contrib/pebble_env/pebble.h

```diff
@@ -52,19 +52,9 @@ typedef enum GCompOp {
 	GCompOpSet
 } GCompOp;
 
-typedef enum GBitmapFormat {
-	GBitmapFormat1Bit = 0,
-	GBitmapFormat8Bit,
-	GBitmapFormat1BitPalette,
-	GBitmapFormat2BitPalette,
-	GBitmapFormat4BitPalette,
-} GBitmapFormat;
-
-struct GBitmap {
-	int dummy;
-};
 
 typedef struct GBitmap GBitmap;
+typedef int GBitmapFormat; // no forward declarations of enums
```

Comment:
The fact that this isn't allowed even in C11 boggles me. Weird. They fixed this in C++11, though, right?